pyproject.toml

```
@@ -106,3 +106,7 @@ initial-content = """
   __version__ = "0.0.0"
   __version_tuple__ = (0, 0, 0)
 """
+[tool.numpydoc_validation]
+checks = ["all", "EX01", "SA01", "ES01" ] # report on all checks, except the following
```

Could you separate these out onto different lines and add a quick explanation of why we are ignoring them for quick reference?
```suggestion
checks = ["all", "EX01", "SA01", "ES01" ] # report on all checks, except the following
```

---

To Greg's comment, I think you could do something like this:

```
# Checks that we chose not to require
checks = [
    "all",
    "EX01",  # No examples section found
    "SA01",  # See Also section not found
    "ES01",  # No extended summary found
]
```